backend/src/routes/books.routes.ts

```diff
@@ -486,56 +486,45 @@ router
    *              properties:
    *                title:
    *                  type: string
+   *                  nullable: false
    *                  example: "작별인사 (김영하 장편소설)"
    *                isbn:
    *                  type: string
+   *                  nullable: false
    *                  example: 9788065960874
    *                author:
    *                  type: string
+   *                  nullable: false
    *                  example: "김영하"
    *                publisher:
    *                  type: string
+   *                  nullable: false
    *                  example: "복복서가"
    *                image:
    *                  type: string
+   *                  nullable: false
    *                  example: "https://bookthumb-phinf.pstatic.net/cover/223/538/22353804.jpg?type=m1&udate=20220608"
    *                categoryId:
    *                  type: integer
+   *                  nullable: false
```

Comment:
integer 타입이 맞을까요?
CreateBookInfo 타입도 그렇고 GET 요청의 반환값은 string이던데 
프론트에서 변환해서 보내드려야 하는 걸까요?

Reply:
그러게요. string으로 수정하겠습니다.
```suggestion
   *                categoryId:
   *                  type: string
   *                  nullable: false
```